docs/COMMANDS.md

```
@@ -171,6 +174,94 @@ Use specops steering
+
+**Claude Code:**
+```
+/specops audit
+/specops audit <name>
+/specops health check
+/specops check drift
+/specops spec health
+```
+
+**Other platforms:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)